cleanlab/multilabel_classification/dataset.py

```diff
@@ -0,0 +1,93 @@
+import pandas as pd
+import numpy as np
+from cleanlab.filter import _find_multilabel_issues_per_class
+from cleanlab.internal.multilabel_utils import get_onehot_num_classes
+from collections import defaultdict
+
+
+def common_multilabel_issues(
+    labels=list,
+    pred_probs=None,
+    *,
+    class_names=None,
```

Comment:
if the docstring is going to call this "Tag", then it should be called `tag_names` here as the variable. 

Otherwise we should keep it as is and change the docstring to call it "class" instead of "tag" everywhere.

Which of these two options do you prefer?

Reply:
We decided to use Class names: https://github.com/cleanlab/cleanlab/blob/6a6c51212e54a38456c33bc7ce5743c824024e78/cleanlab/multilabel_classification/dataset.py#L73